src/scripts/actions/UploaderActions.js

```diff
@@ -87,14 +87,13 @@ export const uploadAndTranscode = (file: Object, videoId: string) => (
     dispatch(
       Notifications.error({
         title: 'Upload Error',
-        message: error,
+        message: 'Something went wrong with your file. May you retry?',
```

Comment:
Nitpick about the verbiage. I think `'Something went wrong with your file. Please try again.'` sounds more natural in english